tests/tests_basics_forwards_cert.yml

```diff
@@ -80,7 +80,7 @@
       include_role:
         name: linux-system-roles.logging
 
-    - include: set_rsyslog_variables.yml
```

Comment:
Maybe use `import_tasks` while you are at it, if you don't mind its limitations? I heard some arguments in favor of imports vs includes. Notably includes have some performance problems, and imports also allow more static analysis (linting) iirc.

Reply:
@pcahyna, is it ok for me to replace all the include_tasks in the tests playbooks? There are more than a dozen...

Reply:
> @pcahyna, is it ok for me to replace all the include_tasks in the tests playbooks? There are more than a dozen...

Not sure - there are slightly different behaviors of import vs. include - I would prefer to keep the `include` behavior for now in this PR.  If we find that using `import` is a significant advantage, we should plan to change all of the roles to use `import` where possible.

Reply:
I think there's no much impact in the replacement. I'm reverting [this second commit](https://github.com/linux-system-roles/logging/pull/152/commits/1d126798c3987f28389ab981cdb514e65dfba2e5) for now.